images/metering-ansible-operator/roles/meteringconfig/tasks/deploy_resources.yml

```diff
@@ -46,6 +46,7 @@
     namespace: "{{ meta.namespace }}"
     definition: "{{ updated_template_results | flatten }}"
     merge_type: ['merge', 'strategic-merge']
+  no_log: true
```

Comment:
Without this, are the secrets being printed? If so..this will make it really difficult to debug other errors.

Reply:
That was a debug artifact I forgot to delete - Updated